testplan/web_ui/testing/src/Toolbar/__tests__/__snapshots__/Toolbar.test.js.snap

```diff
@@ -10,6 +10,11 @@ exports[`Toolbar inserts extra buttons into the toolbar 1`] = `
   >
     <div
       className="filterBox_1kzbnzj"
+      style={
+        Object {
+          "width": undefined,
```

Comment:
why is the with of filterBox is undefined?

Reply:
it is not new, just the width used to be a prop of FilterBox component, and when it was undefined it was not rendered to the dom. Now in style it is shown.  Oh and the rootcause of that was that it was not there in the default props used for the test. I will add it.